src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--28">
+      <div class="calendar__day calendar__day--thu"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29"">
      <div class="calendar__day"></div>
```
Модифікатори повиннні застосовуватись для контейнера. Тоюто початковий день і кількість днів прописуються саме в класі контейнера, а не самого дня